career/ladder.md

```diff
@@ -143,6 +165,12 @@ Here are some examples of other career ladders:
 - [ ] Can debug issues occurring across multiple levels of the stack.
 
 ### Level 4 - Senior Software Engineer
+As a Senior Software Engineer, you consistently show an ability to handle highly
+complex technical challenges across multiple axes without guidance. Your
+technical decisions are always guided by data and based on the highest value for
+all relevant parties. Product managers can rely on you when estimating the scale
+of a larger product, and your track record of turning big challenges into
+successes is apparent to the entire organization.
```

Comment:
Very good!
We should also mention that as a senior software engineer, you've been around for a while. 
```suggestion
As a Senior Software Engineer, you consistently show an ability to handle highly
complex technical challenges across multiple axes without guidance. Your
technical decisions are always guided by data and based on the highest value for
all relevant parties. Your contributions are based on several years of experience where you had the chance to see the long-term effects on solutions you've designed and built. Product managers can rely on you when estimating the scale of a larger product, and your track record of turning big challenges into
successes is apparent to the entire organization.
```

Reply:
Good point. I'll update it.